src/Nancy.Tests/Unit/HeadResponseFixture.cs

```diff
@@ -1,6 +1,10 @@
 namespace Nancy.Tests.Unit
 {
     using System.Collections.Generic;
+    using System.IO;
+

```

Comment:
Unrelated: can we kill this using group whitespace? Preferably add the setting to the .DotSettings-file?